dist/tools/nacl_key_generator/generate-nacl-keys.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
So instead of creating the keys using tweetnacl you prefer to execute an script which runs local tools (if they're installed)?